src/overlays/actors/ovl_Obj_Makeoshihiki/z_obj_makeoshihiki.c

```diff
@@ -6,26 +6,138 @@
 
 #include "z_obj_makeoshihiki.h"
 
+#include "overlays/actors/ovl_Obj_Oshihiki/z_obj_oshihiki.h"
+
+#include <vt.h>
+
 #define FLAGS 0x00000020
 
 #define THIS ((ObjMakeoshihiki*)thisx)
 
 void ObjMakeoshihiki_Init(Actor* thisx, GlobalContext* globalCtx);
 void ObjMakeoshihiki_Draw(Actor* thisx, GlobalContext* globalCtx);
 
-/*
 const ActorInit Obj_Makeoshihiki_InitVars = {
-    ACTOR_OBJ_MAKEOSHIHIKI,
-    ACTORTYPE_PROP,
-    FLAGS,
-    OBJECT_GAMEPLAY_DANGEON_KEEP,
-    sizeof(ObjMakeoshihiki),
-    (ActorFunc)ObjMakeoshihiki_Init,
-    (ActorFunc)Actor_Noop,
-    (ActorFunc)Actor_Noop,
-    (ActorFunc)ObjMakeoshihiki_Draw,
+    ACTOR_OBJ_MAKEOSHIHIKI,       ACTORTYPE_PROP,          FLAGS,
+    OBJECT_GAMEPLAY_DANGEON_KEEP, sizeof(ObjMakeoshihiki), (ActorFunc)ObjMakeoshihiki_Init,
+    (ActorFunc)Actor_Noop,        (ActorFunc)Actor_Noop,   (ActorFunc)ObjMakeoshihiki_Draw,
+};
+
+static BlockConfig sBlocks[] = {
+    {
+        {
+            { 660.0f, 460.0f, 660.0f },
+            { 660.0f, 457.0f, 540.0f },
+            { 780.0f, 454.0f, 540.0f },
+        },
+        0x00,
+        0x00,
+        0x03,
+        0xFF,
+        0x02,
+        0x00,
+        0x0000,
+    },
+    {
+        {
+            { -605.0f, -820.0f, -290.0f },
+            { -365.0f, -905.0f, -290.0f },
+            { -365.0f, -905.0f, -290.0f },
+        },
+        0x00,
+        0x03,
+        0x00,
+        0xFF,
+        0x00,
+        0x00,
+        0x0000,
+    },
```

Comment:
can you play around with removing trailing commas so it doesnt take up so many lines? similar to how our collider init blocks arent a ton of lines

Reply:
Ah yeah, sure thing.

Reply:
I tried a bunch of stuff but could only get it to where it is now..hope that's alright